### ISSUE TITLE
in_array() expects parameter 2 to be array, null given

### ISSUE THREAD

**Issue by K2ouMais** [facade/ignition#82]
I am getting a red error page when going to a localization route like: 
```http://localhost:8000/lang/en```

With that route I just trigger a method like this:
```
class LocalizationController extends Controller
{
    public function index($locale)
    {
        App::setLocale($locale);
        session()->put('locale', $locale);
        return redirect()->back();
    }
}
```
The language cant be set because of the error.

Thanks in advance

**Comment by mpociot** [comment]
Can you share the error with me?

**Comment by K2ouMais** [comment]
![Error](https://user-images.githubusercontent.com/52510455/64237326-68646080-cefc-11e9-92b3-82f559872ead.png)

I have to say that I updated from Laravel 5.8 to 6.0.

**Comment by mpociot** [comment]
Thanks, this is now fixed in 1.4.6 and only occurred when you have published the ignition config from version 1.2.0 👍
